templates/taxi/driver_detail.html

```diff
@@ -22,5 +22,7 @@ <h4>Cars</h4>
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+  <a style="float: left" href="{% url "taxi:driver-licence-update" pk=driver.id %}" class="btn btn-secondary">Update</a>
```

Comment:
The URL name 'taxi:driver-licence-update' should be corrected to 'taxi:driver-license-update' to match the standard American English spelling and ensure consistency with the URL patterns defined in `urls.py`.